pkgs/development/libraries/libffi/default.nix

```diff
@@ -45,6 +47,8 @@ stdenv.mkDerivation rec {
 
   checkInputs = [ dejagnu ];
 
+  nativeBuildInputs = lib.optional stdenv.hostPlatform.isWindows autoreconfHook;
```

Comment:
Can you add a comment why this is needed? I.e. is this a structural problem or will this fix itself in a future libffi release?